java/src/main/kotlin/io/spine/validation/java/util/JavaIdentifiers.kt

```diff
@@ -24,7 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.validation.java
+package io.spine.validation.java.util
```

Comment:
Let's avoid putting more items into `util` package. This file looked OK in `java` package.